src/include/sof/audio/coefficients/asrc/IasSrcFarrow_coeff_24000Hz_to_08000Hz.h

```diff
@@ -0,0 +1,778 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2012 Intel Corporation. All rights reserved.
```

Comment:
2012-2019

and use C comments for .h files (valid for all files in this commit)

Reply:
Yep, done now.